pkg/controller/mongodb/mongodb_controller.go

```diff
@@ -251,15 +277,17 @@ func (r *ReplicaSetReconciler) createOrUpdateStatefulSet(mdb mdbv1.MongoDB) erro
 	return nil
 }
 
-// setAnnotation updates the monogdb resource with the given namespaced name and sets the annotation
+// setAnnotations updates the monogdb resource with the given namespaced name and sets the annotation
 // "key" with the provided value "val"
```

Comment:
Maybe we can rewrite also the rest of the comment? "key" and "val" mean nothing now